src/test/java/io/cryostat/net/openshift/OpenShiftAuthManagerTest.java

```diff
@@ -358,8 +348,7 @@ void shouldNotValidateTokenWithInsufficientPermissions() throws Exception {
                 Matchers.instanceOf(PermissionDeniedException.class));
         PermissionDeniedException pde = (PermissionDeniedException) ExceptionUtils.getRootCause(ee);
         MatcherAssert.assertThat(pde.getNamespace(), Matchers.equalTo(NAMESPACE));
-        MatcherAssert.assertThat(
-                pde.getResourceType(), Matchers.equalTo("recordings.operator.cryostat.io"));
+        MatcherAssert.assertThat(pde.getResourceType(), Matchers.equalTo("pods/exec"));
```

Comment:
I just noticed that with this, we can only check the first resource denied. That means in our mock for `RECORDING` should only specify one k8s resource unless we have some orders. But I think it does not matter that much.

Reply:
I think that's fine. Maybe in the future it's worth refactoring things so that all of the requested permissions are checked and the set of failures are all reported, but the important thing is that unauthorized requests don't get completed.